Patches/CP DOOM & DOOM Kit Classic/CP_DOOM_CE_Patch_Ammo_12Gauge_Dual.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>[CP] DOOM</li>
+			<li>[CP] The DOOM Kit - Classic</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+
+				<li Class="PatchOperationAdd">
+					<xpath>Defs</xpath>
+					<value>
+
+						<!-- ========== Custom variants of the standard 12 Gauge shotgun projectiles, emulating the effect of two ordinary 12 Gauge shotgun shells being fired side-by-side simultaneously ========== -->
+
+						<ThingDef Class="CombatExtended.AmmoDef" ParentName="Base12GaugeBullet">
+							<defName>Bullet_12Gauge_DoubleBarreled_Buck</defName>
+							<label>buckshot pellet</label>
+							<graphicData>
+								<texPath>Things/Projectile/Shotgun_Pellet</texPath>
+								<graphicClass>Graphic_Single</graphicClass>
+							</graphicData>
+							<projectile Class="CombatExtended.ProjectilePropertiesCE">
+								<damageAmountBase>18</damageAmountBase>
+								<pelletCount>18</pelletCount>
+								<armorPenetrationSharp>4</armorPenetrationSharp>
+								<armorPenetrationBlunt>5.7</armorPenetrationBlunt>
+								<spreadMult>17.8</spreadMult>
+							</projectile>
+						</ThingDef>
+
+						<ThingDef Class="CombatExtended.AmmoDef" ParentName="Base12GaugeBullet">
+							<defName>Bullet_12Gauge_DoubleBarreled_Slug</defName>
```

Comment:
If you're doing effectively two shotguns firing at the same time, shouldn't the slug and beanbag rounds have a pelletCount of `2`?

Reply:
Good catch, thanks!

Fixed in 6f83988